### PR TITLE
activate linter check of E275 and clean one file

### PR DESCRIPTION
**Pull request by fchapoton**
this is fixing the last warning for 

E275 missing whitespace after keyword

then cut some lines in the modified file and activate the check for E275 in our linter

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

**Review by kwankyu — APPROVED** [review]
LGTM.

File: src/sage/knots/free_knotinfo_monoid.py [hunk 1]
```diff
@@ -427,10 +434,10 @@ def from_knot(self, knot, unique=True):
                 if unique:
                     return k
                 else:
-                    return[k]  # to be consistent with get_knotinfo
+                    return [k]  # to be consistent with get_knotinfo
```
Comment (dcoudert) [inline src/sage/knots/free_knotinfo_monoid.py]: the `else` could be removed and so indentation reduced.

**Review by dcoudert — COMMENTED** [review]
otherwise LGTM.

File: src/sage/knots/free_knotinfo_monoid.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/knots/free_knotinfo_monoid.py]: ok, fixed

**Review by dcoudert — APPROVED** [review]
LGTM.

**Pull request merged**